src/validations/rules/sar.sch

```diff
@@ -379,7 +379,7 @@
 
             <sch:let
                 name="pl-2-other-than-satisfied-findings"
-                value="oscal:finding/oscal:target[@type = 'objective-id'][oscal:status ne 'satisfied'][contains(@target-id, 'pl-2')]/@target-id" />
+                value="oscal:finding/oscal:target[@type = 'objective-id'][oscal:status ne 'satisfied'][matches(@target-id, '^pl-2$')]/@target-id" />
```

Comment:
Objective target identifiers are not the same as control identifiers. See, e.g., [this](https://github.com/18F/fedramp-automation/blob/master/dist/content/templates/sar/xml/FedRAMP-SAR-OSCAL-Template.xml#L1043).

The PL-2 objectives are
```xml
         <part id="pl-2_obj" name="objective">
            <p>Determine if the organization:</p>
            <part id="pl-2.a_obj" name="objective">
               <prop name="label" value="PL-2(a)"/>
               <p>develops a security plan for the information system that:</p>
               <part id="pl-2.a.1_obj" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(a)(1)"/>
                  <p>is consistent with the organization’s enterprise architecture;</p>
               </part>
               <part id="pl-2.a.2_obj" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(a)(2)"/>
                  <p>explicitly defines the authorization boundary for the system;</p>
               </part>
               <part id="pl-2.a.3_obj" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(a)(3)"/>
                  <p>describes the operational context of the information system in terms of missions and business processes;</p>
               </part>
               <part id="pl-2.a.4_obj" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(a)(4)"/>
                  <p>provides the security categorization of the information system including supporting rationale;</p>
               </part>
               <part id="pl-2.a.5_obj" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(a)(5)"/>
                  <p>describes the operational environment for the information system and relationships with or connections to other information systems;</p>
               </part>
               <part id="pl-2.a.6_obj" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(a)(6)"/>
                  <p>provides an overview of the security requirements for the system;</p>
               </part>
               <part id="pl-2.a.7_obj" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(a)(7)"/>
                  <p>identifies any relevant overlays, if applicable;</p>
               </part>
               <part id="pl-2.a.8_obj" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(a)(8)"/>
                  <p>describes the security controls in place or planned for meeting those requirements including a rationale for the tailoring and supplemental decisions;</p>
               </part>
               <part id="pl-2.a.9_obj" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="INTERVIEW"/>
                  <prop name="method" class="fedramp" value="TEST"/>
                  <prop name="label" value="PL-2(a)(9)"/>
                  <p>is reviewed and approved by the authorizing official or designated representative prior to plan implementation;</p>
               </part>
            </part>
            <part id="pl-2.b_obj" name="objective">
               <prop name="label" value="PL-2(b)"/>
               <part id="pl-2.b_obj.1" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(b)[1]"/>
                  <p>defines personnel or roles to whom copies of the security plan are to be distributed and subsequent changes to the plan are to be communicated;</p>
               </part>
               <part id="pl-2.b_obj.2" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="INTERVIEW"/>
                  <prop name="method" class="fedramp" value="TEST"/>
                  <prop name="label" value="PL-2(b)[2]"/>
                  <p>distributes copies of the security plan and communicates subsequent changes to the plan to organization-defined personnel or roles;</p>
               </part>
            </part>
            <part id="pl-2.c_obj" name="objective">
               <prop name="label" value="PL-2(c)"/>
               <part id="pl-2.c_obj.1" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(c)[1]"/>
                  <p>defines the frequency to review the security plan for the information system;</p>
               </part>
               <part id="pl-2.c_obj.2" name="objective">
                  <prop ns="https://fedramp.gov/ns/oscal"
                        name="response-point"
                        value="You must fill in this response point."/>
                  <prop name="method" class="fedramp" value="EXAMINE"/>
                  <prop name="label" value="PL-2(c)[2]"/>
                  <p>reviews the security plan for the information system with the organization-defined frequency;</p>
               </part>
            </part>
            <part id="pl-2.d_obj" name="objective">
               <prop ns="https://fedramp.gov/ns/oscal"
                     name="response-point"
                     value="You must fill in this response point."/>
               <prop name="method" class="fedramp" value="INTERVIEW"/>
               <prop name="method" class="fedramp" value="TEST"/>
               <prop name="label" value="PL-2(d)"/>
               <p>updates the plan to address:</p>
               <part id="pl-2.d_obj.1" name="objective">
                  <prop name="label" value="PL-2(d)[1]"/>
                  <p>changes to the information system/environment of operation;</p>
               </part>
               <part id="pl-2.d_obj.2" name="objective">
                  <prop name="label" value="PL-2(d)[2]"/>
                  <p>problems identified during plan implementation;</p>
               </part>
               <part id="pl-2.d_obj.3" name="objective">
                  <prop name="label" value="PL-2(d)[3]"/>
                  <p>problems identified during security control assessments;</p>
               </part>
            </part>
            <part id="pl-2.e_obj" name="objective">
               <prop ns="https://fedramp.gov/ns/oscal"
                     name="response-point"
                     value="You must fill in this response point."/>
               <prop name="method" class="fedramp" value="INTERVIEW"/>
               <prop name="method" class="fedramp" value="TEST"/>
               <prop name="label" value="PL-2(e)"/>
               <p>protects the security plan from unauthorized:</p>
               <part id="pl-2.e_obj.1" name="objective">
                  <prop name="label" value="PL-2(e)[1]"/>
                  <p>disclosure; and</p>
               </part>
               <part id="pl-2.e_obj.2" name="objective">
                  <prop name="label" value="PL-2(e)[2]"/>
                  <p>modification.</p>
               </part>
            </part>
         </part>
```